gbfs.md

```diff
@@ -120,41 +136,75 @@ _language_              | Yes         | The language that all of the contained f
   \- name                | Yes         | Key identifying the type of feed this is (e.g. "system_information", "station_information")
   \- url                 | Yes         | Full URL for the feed
 
+
 Example:
 
 ```json
 {
   "last_updated": 1434054678,
   "ttl": 0,
+  "version": "2.1",
   "data": {
     "en": {
       "feeds": [
         {
           "name": "system_information",
-          "url": "https://www.example.com/gbfs/en/system_information"
+          "url": "https://www.example.com/gbfs/2/en/system_information"
         },
         {
           "name": "station_information",
-          "url": "https://www.example.com/gbfs/en/station_information"
+          "url": "https://www.example.com/gbfs/2/en/station_information"
         }
       ]
     },
     "fr" : {
       "feeds": [
         {
           "name": "system_information",
-          "url": "https://www.example.com/gbfs/fr/system_information"
+          "url": "https://www.example.com/gbfs/2/fr/system_information"
         },
         {
           "name": "station_information",
-          "url": "https://www.example.com/gbfs/fr/station_information"
+          "url": "https://www.example.com/gbfs/2/fr/station_information"
         }
       ]
     }
   }
 }
 ```
 
+### gbfs_versions.json
+Each expression of a GBFS feed describes all of the versions that are available.
+
+The following fields are all attributes within the main "data" object for this feed.
+
+Field Name              | Required    | Defines
+------------------------| ------------| ----------
+_versions_              | Yes         | Array that contains one object, as defined below, for each of the available versions of a feed. The array must be sorted by increasing MAJOR and MINOR version number.
+  \- version            | Yes         | String identifying the semantic version of the feed in the form X.Y.
+  \- url                | Yes         | URL of the corresponding gbfs.json endpoint.
+  
+```json
+{
+  "last_updated": 1434054678,
+  "ttl": 0,
+  "version": "2.1",
+  "data": {
+  	"versions":
+		[
+		  {
+			 "version":"1",
+			 "url":"https://www.example.com/gbfs/1/gbfs"
+		  },
+		  {
+			 "version":"2",
+			 "url":"https://www.example.com/gbfs/2/gbfs"
```

Comment:
Just a formatting note, can we add the spaces between the ":" and value here as well? Thanks!